mundane-io/src/test/scala/com/ambiata/mundane/io/LocalTemporarySpec.scala

```diff
@@ -0,0 +1,56 @@
+package com.ambiata.mundane.io
+
+import com.ambiata.disorder._
+import com.ambiata.mundane.control._
+import com.ambiata.mundane.testing.RIOMatcher._
+import com.ambiata.mundane.io.Arbitraries._
+
+import org.specs2._
+
+import scalaz.{Store => _, _}, Scalaz._, effect.IO
+
+class LocalTemporarySpec extends Specification with ScalaCheck { def is = s2"""
+
+ Temporary should clean up its own resources
+ ===========================================
+
+   clean up a file                                   $file
+   clean up a directory                              $directory
+   get a temporary file that exists                  $getFile
+   get a temporary directory that exists             $getDirectory
+   no conflicts                                      $conflicts
+
+"""
+  /** Testing Temporary clean up finalizers */
+  def file = prop((data: String, tmp: LocalTemporary) => for {
+    f <- tmp.fileWithContent(data)
+    e <- Files.exists(f)
+    _ <- RIO.unsafeFlushFinalizers
+    z <- Files.exists(f)
+  } yield e -> z ==== true -> false)
```

Comment:
I'm not 100% sure, but I think these tests are being ignored. You're not implementing `ThrownExceptions` (which you shouldn't), and you're not using `ResultTMatcher`. Let me know if I'm missing something.